.github/workflows/ci.yml

```diff
@@ -351,6 +352,7 @@ jobs:
           AUDIT_CLIENT_TEST_JUSTIFICATION_SUB: '${{ env.WIF_SERVICE_ACCOUNT }}'
           AUDIT_CLIENT_TEST_PROJECT_ID: '${{ env.SERVER_PROJECT_ID }}'
           AUDIT_CLIENT_TEST_BIGQUERY_DATASET: '${{ env.BIGQUERY_DATASET_ID }}'
+          AUDIT_CLIENT_TEST_PRIVATE_KEY_PATH: '${{ github.workspace }}/integration/testrunner/private_key.json'
```

Comment:
what is the usage of the env define above for the private key path, I thought you will use the env here?

Reply:
Updated, removed the env one above.